siriuspy/siriuspy/oscilloscope/keysight.py

```diff
@@ -142,8 +144,9 @@ def wfm_get_data(self, channel=None):
             print('Total acquisition time:', _time.time() - tini)
             # self.send_command(b":WAVeform:STReaming ON\n", get_res=False)
         except Exception:
-            print("Unexpected error:", _sys.exc_info()[0])
-            print('Close connetion by exception')
+            print('Close connection by exception')
+            raise
```

Comment:
same here.

Reply:
Actually, raising the error again changes the interface of the method. This could crash the LI/TB/TS CurrInfo IOCs that use this class. Why did you decide to raise the error again?

Reply:
Maybe in these IOCs we should catch the exception?

Reply:
Actually, it seems the IOC does not use this function

Reply:
Nice, ok then!